### PR TITLE
31 update api

### PR DESCRIPTION
**Pull request by kordano**

### CODE REVIEW

File: bin/run-integrationtests [hunk 1]
```diff
@@ -12,16 +12,14 @@ function test_auth() {
     cp -av test/datahike_server/resources/config.edn resources/config.edn
     ./bin/kaocha --focus datahike-server.integration-test
 }
-
-function test_dev-mode() {
+function test_dev_mode() {
     cp -av test/datahike_server/resources/config-dev-mode.edn resources/config.edn
-    ./bin/kaocha --focus datahike-server.integration-test-dev-mode
+    ./bin/kaocha --focus datahike-server.setup-test
```
Comment (yflim) [inline bin/run-integrationtests:17]: Do the names `test/datahike_server/setup_test.clj`, `setup-test/dev-mode-test`, and `test-dev-mode` make sense together, or should one or more of them be changed?

File: src/clj/datahike_server/server.clj [hunk 2]
```diff
@@ -203,7 +229,7 @@
       (handler request))))
 
 (def route-opts
-  {:reitit.middleware/transform dev/print-request-diffs ;; pretty diffs
+  {;; :reitit.middleware/transform dev/print-request-diffs ;; pretty diffs
```
Comment (yflim) [inline src/clj/datahike_server/server.clj:232]: Just wondering: why was this commented out?

File: test/datahike_server/integration_test.clj [hunk 3]
```diff
@@ -150,11 +139,40 @@
...
+                       :valueType   :db.type/string,
+                       :cardinality :db.cardinality/one,
+                       :id          1}}
            (api-request :get "/schema"
                         {}
+                        {:headers {:authorization "token neverusethisaspassword"
+                                   :db-name       "users"}})))))
+
+(deftest reverse-schema-test
+  (testing "Fetches current schema"
```
Comment (yflim) [inline test/datahike_server/integration_test.clj]: Should be "Fetches current reverse schema" or similar

**Review by yflim — COMMENTED** [review]
Just a couple comments / questions. Looks good otherwise.

File: bin/run-integrationtests (hunk 1 above)
Comment (kordano) [reply bin/run-integrationtests:17]: At some point this needs some better names and potentially just a refactoring into the integration namespace.

File: src/clj/datahike_server/server.clj (hunk 2 above)
Comment (kordano) [reply src/clj/datahike_server/server.clj:232]: For testing and dev there were too many things going on in the repl with that. Might add that through options again.

File: test/datahike_server/integration_test.clj (hunk 3 above)
Comment (kordano) [reply test/datahike_server/integration_test.clj]: Will fix that.

File: bin/run-integrationtests (hunk 1 above)
Comment (kordano) [reply bin/run-integrationtests:17]: I thought a little bit about that and we might add additional tests to the setup test, so things like different backends or env vars. And in the current case we're only testing whether the dev mode works.

File: test/datahike_server/integration_test.clj (hunk 3 above)
Comment (kordano) [reply test/datahike_server/integration_test.clj]: fixed it.

**Pull request merged**